docs/UserGuide.md

```diff
@@ -216,7 +216,14 @@ Example: `p/91234567`
 
 ### `e/EMAIL`
 
-_Use the one from MESSAGE_CONSTRAINTS_
+`EMAIL` should be in the form of `local-part@domain`, where:
+1. `local-part` contains only alphanumeric characters and the special characters `+`, `_`, `.`, `-`, and may not start or end with any special characters.
```

Comment:
Perhaps the numbering should be changed to standard bullet points, since these aren't chronological steps. Same goes for the task part

Reply:
done